tests/gt-examples/03-latex/latex-13-adding_footnotes.Rmd

```diff
@@ -35,7 +35,7 @@ gt(
   data = tbl,
   groupname_col = "date"
 ) %>%
-  tab_header(title = "The Table Title", subtitle = "The subtitle.") %>%
+  #tab_header(title = "The Table Title", subtitle = "The subtitle.") %>%
```

Comment:
Why did you comment that out?

Reply:
I looked into this and if the line is uncommented *and* `tab_options(latex.use.longtable = TRUE)` is removed, we get a LaTeX error when rendering the .Rmd file:

```
! You can't use `\hrule' here except with leaders.
\caption@hrule ->\hrule 
                        \@height \z@ 
l.199 }
        \\ 

Error: LaTeX failed to compile latex-13-adding_footnotes.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See latex-13-adding_footnotes.log for more info.
Execution halted
```

Reply:
@AaronGullickson Are you able to fix this? It is important that LaTeX tables don't get compilation issues if they have a header

Reply:
The error is due to the unnumbered caption being placed within the tabular environment when `tab_options(latex.use.longtable = FALSE)`. I'm not exactly sure how why the error occurs in the test because this specific problem is solved with the code in 727-762 in `R/export.R`.

Reply:
Yes, I must have commented that out early on and forgot about it. However, when I uncomment the line, the Rmd file renders fine for me with the latest changes from the PR. @rich-iannone Can you please confirm it gives you that error with the latest?

Reply:
@AaronGullickson everything is good here now!